mage_ai/tests/data_cleaner/statistics/test_calculator.py

```diff
@@ -23,12 +22,13 @@ def test_calculate_statistics_overview(self):
             [1, None, '2000-07-01 00:00:00+00:00', '5', False],
             ['', None, '899-07-01 00:00:00+00:00', '6', False],
             [3, 'US', None, '7', True],
-        ], columns=['age', 'country', 'date_joined', 'id', 'cancelled'])
+        ],
+                          columns=['age', 'country', 'date_joined', 'id', 'cancelled'])
```

Comment:
the style is weird

Reply:
Not sure how this happened, but resolved now.